ocs_ci/ocs/constants.py

```diff
@@ -1904,13 +1904,13 @@
         "vg-manager_label": "app=vg-manager",
     },
     "default": {
-        "controller_manager_label": "app.kubernetes.io/name=lvm-operator",
-        "topolvm-controller_label": "app.kubernetes.io/name=topolvm-controller",
-        "topolvm-node_label": "app.kubernetes.io/name=topolvm-node",
+        "controller_manager_label": "app.kubernetes.io/name=lvms-operator",
+        "topolvm-controller_label": "app.kubernetes.io/name=topolvm-csi-driver",
+        "topolvm-node_label": "app.kubernetes.io/name=topolvm-csi-driver",
         "vg-manager_label": "app.kubernetes.io/name=vg-manager",
     },
     "411-old": {
-        "controller_manager_label": "app.kubernetes.io/name=lvm-operator",
+        "controller_manager_label": "app.kubernetes.io/name=lvms-operator",
```

Comment:
The change is not backported to 4.11 so I don't think you need to change it here.

Reply:
Done